### PR TITLE
Added playwright test suite

### LINKED ISSUE #146: Initialize Test Suite
Use the implementation at [nearbuilders/devs.near](https://github.com/NEARBuilders/devs.near) for reference.
Introduce playwright.config.js, a test folder, the storage-states, the github workflow, etc. (keep the bos-loader test)

### PR DESCRIPTION
**Pull request by Jikugodwill**
- Resolves #146

### CODE REVIEW

File: playwright-tests/util/bos-loader.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+export async function modifySocialNearGetRPCResponsesInsteadOfGettingWidgetsFromBOSLoader(
```
Comment (itexpert120) [inline playwright-tests/util/bos-loader.js:1]: Can we rename this function smaller? That is way too long.

File: playwright.config.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
+    },
+
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+    },*/
+
+    /* Test against mobile viewports. */
```
Comment (itexpert120) [inline playwright.config.js:79]: Remove comments if not going to use later

File: .github/workflows/continuous-integration.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: CI
```
Comment (elliotBraem) [inline .github/workflows/continuous-integration.yml:1]: With this added, remove the "quality.yml" workflow

File: playwright-tests/util/bos-loader.js (hunk 1 above)
Comment (elliotBraem) [reply playwright-tests/util/bos-loader.js:1]: We can delete this file

File: playwright.config.js (hunk 2 above)
Comment (elliotBraem) [reply playwright.config.js:79]: Keep it, we might

File: playwright.config.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
+// @ts-check
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```
Comment (elliotBraem) [inline playwright.config.js:8]: but this can be removed

**Comment by elliotBraem** [comment]
Can you include issue name in your PR description please #277

**Comment by elliotBraem** [comment]
Looks great! @saadiqbal-dev take a look at this PR, this should be full needs for quickstart.near

**Review by elliotBraem — APPROVED** [review]

**Pull request merged**